frank/utilities.py

```diff
@@ -139,6 +139,46 @@ def jy_convert(x, conversion, bmaj=None, bmin=None):
     return converted
 
 
+def get_fit_stat_uncer(fit):
+    r"""
+    Retrieve a model brightness profile's statistical uncertainty (that due to 
+    the uncertainty on the observed baselines). This is only a lower bound on 
+    the total uncertainty (which also has a systematic contribution due to 
+    sparse sampling of the (u, v) plane).
+
+    Parameters
+    ----------
+    fit : FrankFitter result object
+        Fitted profile to obtain statistical uncertainty of 
+
+    Returns
+    -------
+    sigma: array
+        Statistical uncertainty on the fitted brightness, at the fitted radial
+        points.
+    """
+
+    if 'method' not in fit._info.keys():
+        raise AttributeError("'fit' object lacks '_info.method' key.")
+
+    if fit._info["method"] == "Normal":
+        sigma = np.sqrt(np.diag(fit.covariance))
+
+    elif fit._info["method"] == "LogNormal":
+        # convert stored variance from var(log(brightness)) to var(brightness).
+        # see https://en.wikipedia.org/wiki/Log-normal_distribution
+        log_variance = np.diag(fit.covariance)
+        lin_variance = (np.exp(log_variance) - 1) * \
+            np.exp(2 * np.mean(np.log(fit.I)) + log_variance)
```

Comment:
```suggestion
            np.exp(2 * np.log(fit.I) + log_variance)
```
np.mean here is not correct because it is giving the mean across different radii. The mean provided by wiki would be the posterior mean of log(I) at each radius (assuming that they are independent). 

Note that there is also an argument for neglecting the log_variance term in this line: [see Appendix A.2 here](https://ui.adsabs.harvard.edu/abs/2016A%26A...586A..76J/abstract). I do not have a strong opinion either way here.

Reply:
Sounds good, thanks for catching that and for pointing to the appendix. I've dropped the log_variance term in [67343a7](https://github.com/discsim/frank/pull/201/commits/67343a75d67b750e9362765542ead24ecd432c7c)